UnitTestProject/PlayerTest.cs

```diff
@@ -247,64 +246,93 @@ public void Compare_Save_Files()
             );
             #endregion
 
-            #endregion
             SaveManager.Overwrite(newSave, 0);
 
             SaveData loadedData = SaveManager.GetSave(0);
 
-            #region Fill The New Save File
-            saveName = "First Save";
-            activeScene = 2;
+            #region Assert On Save Values
+            if (newSave.BuildVersion != loadedData.BuildVersion)
+                Assert.Fail("Build versions are not the same");
 
-            playerName = "Red";
-            trainerID = 55323;
-            secretID = 64123;
-            isMale = false;
+            if (newSave.SaveName != loadedData.SaveName)
+                Assert.Fail("SaveNames are not the same");
 
-            pokedex = new bool?[] { null, false, true, false, null };
-            playerTime = new TimeSpan(4, 20, 53);
+            if (newSave.TimeCreated != loadedData.TimeCreated)
+                Assert.Fail("TimeCreated are not the same");
 
-            playerPosition = new SerializableVector3(0, 1, 0);
-            playerDirection = 2;
-            followerPosition = new SerializableVector3(1, 0, 0);
-            followerDirection = 1;
+            if (newSave.PlayerName != loadedData.PlayerName)
+                Assert.Fail("PlayerNames are not the same");
 
-            playerParty = new Pokemon[]
-            {
-                new Pokemon(Pokemons.CRANIDOS),
-                new Pokemon(Pokemons.UMBREON),
-                new Pokemon(Pokemons.TURTWIG)
-            };
-            playerPC = new Pokemon[,]
```

Comment:
For player pc, it should be written as: [number of boxes player has, number of Pokemons that can fit in one box], by default all values should be "pokemon.none" except for where there IS a Pokemon.

Reply:
This has been solved. New PC's are now created in PlayerTest as playerPC = new Pokemon[,]
All null spaces are filled with Pokemon.NONE and some are filled with Pokemon.